RealmSwift/Combine.swift

```diff
@@ -486,10 +486,12 @@ public enum Publishers {
             self.realm = realm
         }
 
-        /// Allows you to capture the `NotificationToken` produced by observing a Realm.
+        /// Captures the `NotificationToken` produced by observing a Realm Collection. This allows you to do notification skipping
```

Comment:
Paragraph break after the first sentence, as when the docstrings are rendered the first line is used as the short description and then everything after that is formatted a bit differently.